MarvinLee_Portfolio_FEND/css/main.css

```diff
@@ -1,8 +1,7 @@
-body, html {
-  margin: 0;
-  padding: 0;
-  width: 100%;
-  height: 100%;
+body {
+  max-width: 950px;

```

Comment:
We don't want the page to be larger than 950px. Feel free to change with to what you like

Reply:
Usaually the web site is not lager than 950px?